app/controllers/profiles_controller.rb

```diff
@@ -2,5 +2,8 @@ class ProfilesController < ApplicationController
   def profile
     redirect_to '/users/sign_in' unless user_signed_in?
     @user = current_user
+    @posts = current_user.posts
+    @comments = current_user.comments
+    @total_recieved_spice = @posts.to_a.inject(0) { |s, p| s + p.votes_for.size } + @comments.to_a.inject(0) { |s, c| s + c.votes_for.size }
```

Comment:
`@posts.map { |p| p.votes_for.size }.reduce(&:+) + @comments.map { |c| c.votes_for.size }.reduce(&:+)`

Reply:
actually just `.reduce(:+)` works, you don't need the &